how-to/networking/dnssec-troubleshooting.md

```diff
@@ -105,107 +104,140 @@ Let's unpack this answer for the important troubleshooting parts:
 
 But to bring the responsibility of doing DNSSEC validation to the tool itself, we use the `+cd` flag in our queries, to tell the resolver to not attempt that validation. Otherwise we will just get back a generic `SERVFAIL` error:
 
-    $ delv @127.0.0.1 -t A +dnssec +multiline dnssec-failed.org
+    $ delv @127.0.0.53 -t A +dnssec +multiline dnssec-failed.org
     ;; resolution failed: SERVFAIL
 
 With the `+cd` flag present, however, `delv` itself will do the validation. It will fail again, but now with a DNSSEC-specific error:
 
-    $ delv @127.0.0.1 -t A +dnssec +cd +multiline dnssec-failed.org
+    $ delv @127.0.0.53 -t A +dnssec +cd +multiline dnssec-failed.org
     ;; validating dnssec-failed.org/DNSKEY: no valid signature found (DS)
     ;; no valid RRSIG resolving 'dnssec-failed.org/DNSKEY/IN': 127.0.0.1#53
     ;; broken trust chain resolving 'dnssec-failed.org/A/IN': 127.0.0.1#53
     ;; resolution failed: broken trust chain
 
 If needed, `delv` can be told to not perform DNSSEC validation at all, by passing the `-i` flag. Together with the `+cd` flag, which instructs the Validating Resolver to not perform validation either, we get this result:
 
-    $ delv @127.0.0.1 -i -t A +dnssec +cd +multiline dnssec-failed.org
+    $ delv @127.0.0.53 -i -t A +dnssec +cd +multiline dnssec-failed.org
     ; answer not validated
     dnssec-failed.org.      100 IN A 96.99.227.255
 
 For a good DNSSEC domain, `delv` will return a validated answer:
 
-    $ delv @127.0.0.1 -t A +multiline +cd isc.org
+    $ delv @127.0.0.53 -t A +multiline +cd isc.org
     ; fully validated
-    isc.org.                300 IN A 151.101.2.217
-    isc.org.                300 IN A 151.101.66.217
-    isc.org.                300 IN A 151.101.130.217
-    isc.org.                300 IN A 151.101.194.217
-    isc.org.                300 IN RRSIG A 13 2 300 (
-                                    20241107074911 20241024070338 27566 isc.org.
-                                    BIl7hov5X11CITexzV9w7wbCOpKZrup3FopjgF+RIgOI
-                                    5A8p8l2dJCLp/KBn/G6INj7TOHTtrGs1StTSJVNksw== )
+    isc.org.		299 IN A 151.101.2.217
+    isc.org.		299 IN A 151.101.66.217
+    isc.org.		299 IN A 151.101.130.217
+    isc.org.		299 IN A 151.101.194.217
+    isc.org.		299 IN RRSIG A 13 2 300 (
+    				20250908112301 20250825104017 27566 isc.org.
+    				9oclTno0Ub2NmUEXdyLv0zqwPBbbUVmT3RX4aP4BQQ+h
+    				4g839JXuCKHufXSPkWh/GJe/MveP83dDvJkrMmEIzg== )
+
+Given that above we used the `+cd` flag, this means that the validation was done by `delv` itself. We will get the same result without that flag if the resolver also succeeds in the DNSSEC validation, and provides an answer.
+
+## Client-side tooling: resolvectl
+
+The local stub resolver systemd-resolved, can perform DNSSEC validation locally, using its high-level `resolvectl` tool. The local cache and state about DNS servers should be reset in systemd-resolved, to get reliable results.
```

Comment:
Superfluous commas in "systemd-resolved," and "locally,"? Or perhaps one missing after "stub resolver,"?